media/js/newsletter/country.es6.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    clearFormErrors,
+    disableFormFields,
+    enableFormFields,
+    postToBasket
+} from './form-utils.es6';
+
+let _form;
+let _countrySelect;
```

Comment:
**Curious**: why use the underscore here?

Reply:
@nathan-barrett this is mostly a stylistic thing, to try and convey the variable in private to the ES module and is lexically scoped.